src/track/trace_libunwind.cpp

```diff
@@ -60,9 +61,11 @@ void Trace::print()
 void Trace::setup()
 {
     // configure libunwind for better speed
+#if UNW_CACHE_PER_THREAD
     if (unw_set_caching_policy(unw_local_addr_space, UNW_CACHE_PER_THREAD)) {
         fprintf(stderr, "WARNING: Failed to enable per-thread libunwind caching.\n");
     }
+#endif
 #if LIBUNWIND_HAS_UNW_SET_CACHE_SIZE
```

Comment:
this here is set via `cmake/FindLibunwind.cmake` - you'll have to add the same check for `UNW_CACHE_PER_THREAD` and then add it to `libunwind_config.h.cmake`